modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -48,37 +69,42 @@
         (qp/process-query query))
       @native-query)))
 
+(defn- sql->lines [sql]
+  (str/split-lines (mdb.query/format-sql sql :redshift)))
+
 (deftest remark-test
-  (testing "single field user-specified value"
-    (let [expected (str/replace
-                    (str
-                     "-- /* partner: \"metabase\", {\"dashboard_id\":5678,\"chart_id\":1234,\"optional_user_id\":1000,"
-                     "\"optional_account_id\":\"" (public-settings/site-uuid) "\","
-                     "\"filter_values\":{\"id\":[\"1\",\"2\",\"3\"]}} */"
-                     " Metabase:: userID: 1000 queryType: MBQL queryHash: cb83d4f6eedc250edb0f2c16f8d9a21e5d42f322ccece1494c8ef3d634581fe2\n"
-                     "SELECT \"%schema%\".\"test_data_users\".\"id\" AS \"id\","
-                     " \"%schema%\".\"test_data_users\".\"name\" AS \"name\","
-                     " \"%schema%\".\"test_data_users\".\"last_login\" AS \"last_login\""
-                     " FROM \"%schema%\".\"test_data_users\""
-                     " WHERE (\"%schema%\".\"test_data_users\".\"id\" = 1 OR \"%schema%\".\"test_data_users\".\"id\" = 2"
-                     " OR \"%schema%\".\"test_data_users\".\"id\" = 3)"
-                     " LIMIT 2000")
-                    "%schema%" redshift.test/session-schema-name)]
-      (mt/test-driver :redshift
+  (testing "if I run a Redshift query, does it get a remark added to it?"
+    (mt/test-driver :redshift
+      (let [expected (for [line ["-- /* partner: \"metabase\", {\"dashboard_id\":5678,\"chart_id\":1234,\"optional_user_id\":1000,\"optional_account_id\":\"{{site-uuid}}\",\"filter_values\":{\"id\":[\"1\",\"2\",\"3\"]}} */ Metabase:: userID: 1000 queryType: MBQL queryHash: cb83d4f6eedc250edb0f2c16f8d9a21e5d42f322ccece1494c8ef3d634581fe2"
```

Comment:
I didn't change the query itself in most of these tests, but I did rework a bunch of them to format the SQL and split it by line. You can get nice diffs with an array of lines that you can't get with one giant SQL string. And it's a lot easier to read too I think